client/src/components/FormSignUp.jsx

```diff
@@ -1,26 +1,95 @@
-import React from 'react';
+import React, {useState} from 'react';
 
 function FormSignUp() {
+  const [username, setUsername] = useState("");
+  const [age, setAge] = useState("");
+  const [sex, setSex] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+
+  const handleUsername = (e) => {
+    setUsername(e.target.value);
+  };
+
+  const handleAge = (e) => {
+    setAge(e.target.value);
+  };
+
+  const handleSex = (e) => {
+    setSex(e.target.value);
+  };
+  const handleEmail = (e) => {
+    setEmail(e.target.value);
+  };
+
+  const handlePassword = (e) => {
+    setPassword(e.target.value);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    alert("User Added")
+    const userData = {
+      username: username,
+      age: age,
+      sex: sex,
+      email: email,
+      password: password,
+    };
+
+   try{
+    const add = await fetch("http://localhost:3001/users", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        "Access-Control-Allow-Origin": "http://localhost:3001"
```

Comment:
`"Access-Control-Allow-Origin": "http://localhost:3001"` this line is trying to resolve the CORS problem but not working.